setup.cfg

```diff
@@ -77,6 +77,16 @@ install_requires =
   # NOTE: per issue #509 0.15.34 included in debian backports
   typing-extensions; python_version < "3.8"
 
+[options.extras_require]
+test =
+  pytest >= 6.0.1
+  pytest-cov >= 2.10.0
+  # https://github.com/pytest-dev/pytest-cov/issues/422
+  pytest-xdist >= 1.34.0,<2.0.0
+  # Needed to avoid DeprecationWarning errors in pytest:
+  setuptools >= 40.4.3
+  wheel >= 0.35.0
```

Comment:
This is weird.

Reply:
Not sure what is weird, clearly that was not really an actionable remark.

Reply:
runtime dependency on wheel/setuptools (even optional) is weird.